source/dub/compilers/buildsettings.d

```diff
@@ -138,20 +132,7 @@ private:
 
 	static void prepend(ref string[] arr, in string[] vals, bool no_duplicates = true)
```

Comment:
Just noting that the vibe.d style guide, on which the code is still based, uses `under_score` notation for parameters to distinguish them from public non-type symbols. Since this is not part of the public API, though, I don't really mind it staying like it is in the PR now. One day we can decide on the Phobos vs. vibe.d style issue as a whole.